### PR TITLE
fix: sort deployment by last update time

### PR DESCRIPTION
**Pull request by wansir**
Signed-off-by: hongming <talonwan@yunify.com>

**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

1. sort deployment by last update time.
2. sort pod by start time.

### CODE REVIEW

File: pkg/models/resources/pods.go [hunk 1]
```diff
@@ -221,6 +221,14 @@ func (*podSearcher) fuzzy(fuzzy map[string]string, item *v1.Pod) bool {
...
 func (*podSearcher) compare(a, b *v1.Pod, orderBy string) bool {
 	switch orderBy {
+	case StartTime:
+		if a.Status.StartTime == nil {
+			return true
+		}
+		if b.Status.StartTime == nil {
+			return false
+		}
+		return a.Status.StartTime.Before(b.Status.StartTime)
```
Comment (runzexia) [inline pkg/models/resources/pods.go]: if a is older than b. `true`
if a is not started . `true`
if a is started and b is not started `false` / / If b is not started, it should be false？

Comment (wansir) [reply pkg/models/resources/pods.go]: fixed

**Pull request merged**